redirect.go

```diff
@@ -0,0 +1,19 @@
+package router
+
+import (
+	"github.com/gopherjs/gopherjs/js"
+	"github.com/gopherjs/vecty"
+)
+
+// Redirect ...
+func Redirect(route string) EventCallback {
```

Comment:
I don't think you need to return `EventCallback` here. You can simply just run the `js.Global.Get("history").Call("pushState"...)` and let the browser redirect you. 

Also now that we have this, the `onClick` handler can just call `Redirect` instead of duplicating the code :)